src/components/Identity.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+
+interface IdentityProps {
+  credential?: __esri.Credential;
+  handleLogin: () => void;
+  handleLogout: () => void;
+}
+
+export const Identity = (props: IdentityProps) => {
+  if (props.credential) {
+    return (
+      <span>
+        Logged in as {props.credential.userId} 
+        <button onClick={props.handleLogout}>
+          Logout
+        </button>
+      </span>
+    (;
+  } else {
+    return (
+      <button onClick={props.handleLogin}>
+        Login
+      </button>
+    );
+  }
```

Comment:
I broke up these lines to make then less long.

Reply:
:white_check_mark: